src/content/news/2025-09-04-maplibre-newsletter-august-2025/index.mdx

```diff
@@ -0,0 +1,258 @@
+
+## 📱 MapLibre Native
+
+- **Want to help test MapLibre Android?** In addition to our existing continuous integration tests we are taking a crowdsourced approach to testing. We now make releases of the [MapLibre Android TestApp](https://github.com/maplibre/maplibre-native/releases?q=android-testapp&expanded=true). If you want to help, simply download the apk and install it on a Android device you have laying around. Next, open the 'Long running Maps' activity and let it run for up to 72 hours! Crashes are logged to our [Sentry](https://sentry.io/welcome/) so that MapLibre developers can investigate and fix issues.
+
+<div style={{ textAlign: "center" }}>
+  <Image
+    src={nativeTestApp}
+    style={{
+      maxWidth: "300px",
+      maxHeight: "600px",
+      width: "100%",
+      height: "auto",
+      display: "inline-block",
+    }}
+    alt="Android TestApp running the long running test (or 'Desert Bus')"
+  />
+  <p
+    style={{
+      fontSize: "0.9rem",
+      color: "#aaa",
+      marginTop: "8px",
+      fontStyle: "italic",
+    }}
+  >
+    <em>Android TestApp running the long running test (or 'Desert Bus')</em>
+  </p>
+</div>
+
+- MapLibre Android 11 was released well over a year ago, but some people are still on MapLibre Android 10.x.x. We made a [release](https://github.com/maplibre/maplibre-native/releases/tag/android-v10.3.5) for this previous major release to bring 16 KB pages support since Google will [require](https://android-developers.googleblog.com/2025/05/prepare-play-apps-for-devices-with-16kb-page-size.html) it for apps on the Google Play store starting November 1.
+
+- Two minor regular releases [MapLibre Android 11.13.1](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.13.1) and [MapLibre iOS 6.18.1](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.18.1).
+
+- [MapLibre Node.js](https://github.com/maplibre/maplibre-native/releases/tag/node-v6.2.0) also had a release that fixes a problem where it would not render more than 32 frames.
+
+## 🌐 MapLibre Web
+
+Version [5.6.2](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.6.2) and [5.7.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.7.0) were released this month.
+
+These releases focused mainly on some bugs fixes, especially related to the recently added [`global-state`](https://maplibre.org/maplibre-gl-js/docs/API/classes/Map/#setglobalstateproperty) which probably still needs more ironing out. There was also an addition of the [`coveringTiles`](https://maplibre.org/maplibre-gl-js/docs/API/classes/Map/#coveringtiles) method to the public API after it was buried a bit too deep when adding the globe support in version 5.
+
+Huge thanks to all contributors who helped make these improvements possible, we appreciate your efforts and look forward to more contributions!
+
+## 🧩 MapLibre Tile Format
```

Comment:
This sounds like it's just another month, but I think we should make it clear that we are getting ready for the 1.0 release.

Suggest to replace this section with:

---

We are finalizing the specification for the 1.0 release of the [maplibre-tile-spec](https://github.com/maplibre/maplibre-tile-spec) next month. We are also preparing the Java based MVT to MLT converter for release. Last-minute comments are still possible in [this thread](https://github.com/maplibre/maplibre-tile-spec/issues/519).

Reply:
thanks for the pointers, modifying the section.